medical-clinic/src/main/java/com/Mafiuz04/medicalclinic/controller/PatientController.java

```diff
@@ -18,13 +19,13 @@ public class PatientController {
     private final PatientService patientService;
 
     @GetMapping()
-    public List<PatientDto> getPatients() {
-        return patientService.getPatients();
+    public List<PatientDto> getPatients(Pageable pageable) {
+        return patientService.getPatients(pageable);
     }
 
-    @GetMapping("/{email}")
-    public PatientDto getPatientByEmail(@PathVariable String email) {
-        return patientService.getPatientByEmail(email);
+    @GetMapping("/{id}")
```

Comment:
jak zmienisz to zmien tez nazwe metody -> m  public PatientDto getPatientByEmail(@PathVariable Long id) { -> chyba teraz byId